### ISSUE TITLE
Extending `search.separator` seems not to work

### ISSUE THREAD

**Issue by scherermichael** [squidfunk/mkdocs-material#1755]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

`search.separator` seems not to work in version 5 (tested with 5.2.3 and 5.0.0). I extended it with `'[\s\-/\\]+'` in order to allow searching for folders and filenames. But this change is not reflected in the search results.

With version 4.6.3 the same search configuration works as expected. The default MkDocs template works as well. (See https://github.com/scherermichael/mkdocs-material-separator/blob/master/README.md for how I tested those versions.)

### Expected behavior

After extending the separator setting, file paths (Windows or Linux style) should appear in search results if you search for a folder or file.

### Actual behavior

No file path shows up in the result.

### Steps to reproduce the bug

1. Clone test repository containing a small sample `index.md`:

    ```sh
    git clone git@github.com:scherermichael/mkdocs-material-separator.git
    ```

    or

    ```sh
    git clone https://github.com/scherermichael/mkdocs-material-separator.git
    ```

2. In the repo's root directory, start `squidfunk/mkdocs-material:5.2.3`:

    ```sh
    docker run --rm -it -p 8000:8000 -v "${PWD}/v5:/docs" squidfunk/mkdocs-material:5.2.3
    ```

3. Open <http://localhost:8000> and enter the following search terms one by one:

    - `ProgramData`
    - `PLOSSYS`
    - `config`
    - `opt`  --> This one works!
    - `seal`
    - `etc`

### Package versions

* Python: `python --version`

    Python 3.8.1

* MkDocs: `mkdocs --version`
  
    mkdocs, version 1.1.2 from /usr/local/lib/python3.8/site-packages/mkdocs (Python 3.8)

* Material: `pip show mkdocs-material | grep -E ^Version`

    Version: 5.2.3

### Project configuration

``` yaml
site_name: Search Test v5
site_url: "http://localhost:8000"
theme:
  name: material

plugins:
  - search:
      separator: '[\s\-/\\]+'
      lang:
        - en

nav:
  - index.md
```

### System information

* OS: macOS 10.15.5
* Browser: Chrome 83.0.4103.97 (Official Build) (64-bit)

**Comment by squidfunk** [comment]
I checked the code base and – you’re right! It was simply forgotten and nobody noticed until now. Here’s the line in question:

https://github.com/squidfunk/mkdocs-material/blob/de20d673ddafeb021327bedcf167ca8592a7c7ae/src/assets/javascripts/components/Material/Search/Result.jsx#L147

Im currently on a short vacation and will provide a fix as soon as I get my hands on a notebook again. Probably the beginning of next week.

**Comment by scherermichael** [comment]
Great! I'm glad it's easy to fix. Thanks for the swift response and have a nice vacation.

**Comment by squidfunk** [comment]
I found some time to add the line in question, `master` should work now. New release on Monday.

**Comment by squidfunk** [comment]
Released as part of 5.3.0.
